lib/SPIRV/libSPIRV/spirv.hpp

```diff
@@ -71,21 +71,18 @@ enum ExecutionModel {
     ExecutionModelFragment = 4,
     ExecutionModelGLCompute = 5,
     ExecutionModelKernel = 6,
-    ExecutionModelCount, /* internal use only */
```

Comment:
Well if we can have ExecutionModelCount officially defined in the spirv.hpp the problem is solved. :-) I am wondering how the spirv-tools check the validity of ExecutionModel.